packages/blade/src/components/Indicator/Indicator.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Title, Subtitle, Primary, ArgsTable, Stories, PRIMARY_STORY } from '@storybook/addon-docs';
+import type { ReactElement } from 'react';
+import { Highlight, Link } from '@storybook/design-system';
+
+import type { IndicatorProps } from './Indicator';
+import { Indicator as IndicatorComponent } from './Indicator';
+import useMakeFigmaURL from '~src/_helpers/storybook/useMakeFigmaURL';
+import Box from '~components/Box';
+import { Button } from '~components/Button';
+import { getPlatformType } from '~utils';
+
```

Comment:
you might need to incorporate the storywrapper and sandpack changes as Saurabh's PR is merged?

Reply:
Done in the refactor PR https://github.com/razorpay/blade/pull/830/commits/21df0fb87b7a8bfa523fc8fbfa81a72056d46efb